test/test-functions

```diff
@@ -99,11 +103,20 @@ run_qemu() {
         exit 1
     fi
 
-    KERNEL_APPEND="root=/dev/sda1 \
+if [[ "$LOOKS_LIKE_SUSE" ]]; then
+    UUID=$(blkid -o value -s UUID "${LOOPDEV}p1")
```

Comment:
Thanks for the review. We can omit this and use /dev/sda1 directly. I have updated my patch accordingly.